app/views/file_manager/index.html.erb

```diff
@@ -0,0 +1,94 @@
+<% @title = "File Manager" %>
```

Comment:
I think it would be helpful if clicking the file manager tag redirected back to the file manager. It's a little bit cumbersome to use the .. option to navigate backwards

Reply:
Created this too